index.js

```diff
@@ -97,7 +97,7 @@ SassCompiler.prototype._nativeCompile = function(source, callback) {
       callback(null, data);
     }),
     error: (function(error) {
-      callback(error);
+      callback(error.message);
```

Comment:
util.inspect isn't quite as readable, so I'll leave it up to you which you prefer:
```
12 Mar 16:15:46 - error: Compiling of 'app/styles/dev.scss' failed. { status: 1,
  file: 'app/styles/helpers/_functions.scss',
  line: 6,
  column: 11,
  message: 'argument `$map` of `map-get($map, $key)` must be a map\nBacktrace:\n\tapp/styles/helpers/_functions.scss:6, in function `map-get`\n\tapp/styles/helpers/_functions.scss:6, in function `palette`\n\tapp/styles/base/_bootstrap-override.scss:4',
  code: 1 }
```